### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

**Review by dmitry-eliseev-devexpress — COMMENTED** [review]
**Structure and Semantics**

This is a simple how-to readme topic for a code example. The topic serves its purpose in general, however, certain sentences raised more questions than answered for me. Refer to the corresponding comments for details. We need to clarify these places before we'll be able to move forward.

**Images**

The image complies with DevExpress image guidelines and serves its purpose. Well done.

**Language**

I like your disciplined approach to writing -- you definitely nailed down the basics in your head. Well done. However, you need to pay more attention to consistency, advanced article usage cases, and technical details, of course. As I mentioned before, certain sentences did not work for me. Please clarify.

**Bottom Line**

This is a good topic but it needs some more love before I can approve it and we go further down the pipeline.

**Review by dmitry-eliseev-devexpress — APPROVED** [review]

**Review by dirk-pieterse — APPROVED** [review]
